app/templates/direct-award/save-search.html

```diff
@@ -30,21 +30,25 @@
 <div class="single-question-page save-search-page">
   <div class="grid-row">
     <div class="column-one-whole">
-        {% if form.name.errors or save_search_selection_error %}
+        {#
+          we cannot use the toolkit/forms/validation.html template here,
+          as this page was designed with a govuk style error summary banner
```

Comment:
As discussed, it's a shame that this version doesn't have the yellow focus border on the div for keyboard users, but given the difference in user experience (displaying message rather than question name) we should get some input from Content before we change it. But we should change it at some point 😸

Reply:
Yes, the accessibility issues do concern me though...